### ISSUE TITLE
Is the cheat still alive?

### ISSUE THREAD

**Issue by ddjem87** [Teklad/csgo-horker#68]
I know teklad its busy af but there is no new info about the cheat or any thing new on this version

we only want to know did you guys stop working on the cheat or just taking a break from it.

**Comment by Teklad** [comment]
Its moving along.  I'm programming around my work schedule and my work schedule is very hectic.  Hoping to run into some more holiday time soon so I can get this finished.

**Comment by lostinblue** [comment]
Yes, several of the memory offsets and buffer sizes have been updated and fixed this. Would be best to compile and test to confirm. If your question is answerd would be best to close and specify the answer that worked for you.
